packages/components/src/components/dropdown/dropdown.stories.ts

```diff
@@ -0,0 +1,303 @@
+import '../../solid-components';
+import { html } from 'lit';
+import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
+import { userEvent } from '@storybook/testing-library';
+import { waitUntil } from '@open-wc/testing-helpers';
+import { withActions } from '@storybook/addon-actions/decorator';
+
+const { argTypes, parameters } = storybookDefaults('sd-dropdown');
+const { generateTemplate } = storybookTemplate('sd-dropdown');
+const { overrideArgs } = storybookHelpers('sd-dropdown');
+
+export default {
+  title: 'Components/sd-dropdown',
+  component: 'sd-dropdown',
+  args: overrideArgs([
+    { type: 'slot', name: 'trigger', value: '<sd-button slot="trigger">Trigger</sd-button>' },
+    {
+      type: 'slot',
+      name: 'default',
+      value: '<div class="slot slot--border slot--background"></div>'
+    },
+    {
+      type: 'attribute',
+      name: 'open',
+      value: true
+    }
+  ]),
+  argTypes,
+  parameters: { ...parameters },
+  decorators: [
+    withActions,
+    (story: any) =>
+      html`<style>
+          div.slot {
+            width: 24px;
+            height: 24px;
+          }
+          sd-dropdown:not([rounded]) sd-button::part(base),
+          sd-dropdown:not([rounded]) .slot {
+            border-radius: 0;
+          }
+          td.template,
+          .example,
+          div.example {
+            width: 120px;
+            height: 70px;
+          }
+          .example {
+            display: flex;
+            flex-direction: column;
+            padding: 12px;
+          }
+          .template-placement sd-dropdown[placement^='top'] .slot,
+          .template-placement sd-dropdown[placement^='bottom'] .slot {
+            width: 48px;
+          }
+
+          .template-placement sd-dropdown[placement^='left'] .slot,
+          .template-placement sd-dropdown[placement^='right'] .slot {
+            height: 36px;
+          }</style
+        >${story()}`
+  ] as unknown
+};
+
+/**
+ * Default: This shows sd-dropdown in its default state.
+ */
+
+export const Default = {
+  render: (args: any) => {
+    return html`<div style="height: 200px; overflow: hidden;">
+      ${generateTemplate({
+        constants: {
+          type: 'slot',
+          name: 'default',
+          value: '<div class="example"></div>'
+        },
+        args
+      })}
+    </div>`;
+  }
+};
+
+/**
+ * For round triggers, set the `rounded` attribute. this automatically adds a minimal distance of 1px between the trigger and the panel.
+ */
+
+export const Rounded = {
```

Comment:
is it supposed to be that the trigger is rounded as well?
it feels confusing as this is nothing we provide but control it with this prop.

Reply:
It is meant for "round" triggers. Therefore this story shows best practice. Showing with a non-round trigger would be wrong. As soon as we have the sd-navigation-item, we can use it instead of the non-rounded-element button.

I'll create a follow-up ticket or add it to the sd-navigation-item ticket!

Reply:
if its about the rounded triggers, why is the flyout rounded as well? the comment and the shown sample feels confusing.

Reply:
When triggers are rounded, dropdowns have to be rounded, too.

The example shows a rounded trigger and therefore the `rounded` attribute has to be set to true. Ideas how to improve the docs?